native/cocos/platform/android/java/src/com/cocos/lib/CocosHelper.java

```diff
@@ -427,4 +428,8 @@ public void run() {
             }
         });
     }
+    public static boolean supportHPE() {
+        PackageManager pm = GlobalObject.getActivity().getPackageManager();
```

Comment:
Why not use `GlobalObject.getContext()` ?
We need to use `getContext()` if we can.

Reply:
The method is not present in the `CocosHelper` class and can be accessed through the `GlobalObject`.

Reply:
While both `getContext` and `getActivity` can be used interchangeably here, using `getContext` would be a better option if we plan to support `Activityless` mode in the future.